master/internal/task/preemptible/preemptible.go

```diff
@@ -84,7 +85,12 @@ func (p *Preemptible) Preempt(timeoutCallback TimeoutFn) {
 	if !p.preempted {
 		p.wg.Go(func(ctx context.Context) {
 			// don't acquire a lock in here without changing close to not lock while it waits.
-			t := time.NewTimer(DefaultTimeout)
+			timeout := DefaultTimeout
+			if debugTimeout := config.GetMasterConfig().InternalConfig.PreemptionTimeout; debugTimeout != nil {
```

Comment:
why not make the config a `model.Duration`?

Reply:
I was gonna do that but then I saw ```TensorboardTimeout```

I'll switch it to a duration since ```TensorboardTimeout``` isn't a good example to do